contract.md

```diff
@@ -12,29 +23,77 @@ like to work on, as part of your project (for example, other things related to D
 you have come across, or topics from other classes)? Why are you interested in working on
 these things?_
 
+Calling back to one of the earliest exercises in this class, I'd like to work on
+creating a product not just for me, but for the intended audience. That is, I
+should be continiously checking in with other people to see if what intuitively
+makes sense to me also makes sense to them, and seeing if there are any features
+that I'm missing that they would fine helpful (among other things). I feel like
+when I work on projects, I tend to tunnel-vision on what I want and not others,
+so it would be good to work on that.
+
+This is kind of a stretch goal but I'd also want to work on building a GUI that
+isn't webdev based, just because I've already done a lot of webdev. However,
+this isn't that related to DSLs, hence why I say it's a stretch goal; it might
+be more efficient to just work with what I already know.
```

Comment:
Yeah, I'd say with the limited time we have and the intended focus on language design that webdev would be the way to do. But it's a good stretch goal and there's lots of interesting libraries out there - if you end up looking for one, I've used wxwidgets in C++ before for a project and it wasn't terrible to pick up.